### DISCUSSION TITLE
خرید اشتراک از طریق رمز ارزها

### DISCUSSION THREAD

**Discussion by PaulDorf90** [getlantern/lantern-forum-fa#8] [General]
لطفا درگاه خرید اشتراک لنترن با ارزهای دیجیتال را فراهم کنید چون کسانی که در داخل ایران هستن هی راه دیگه ای برای پرداخت و تهیه اشتراک نخواهند داشت 
با تشکر از شما

**Comment by jak3344** [comment]
سلام اره واقعا باید این شرایط فعال بشه مثلا با تتر بشه پرداخت اشتراک انجام داد
